.github/workflows/pkgdown.yaml

```diff
@@ -8,46 +8,50 @@ name: pkgdown
 
 jobs:
   pkgdown:
-    runs-on: macOS-latest
+    runs-on: ubuntu-18.04
```

Comment:
The new workflows use ubuntu for the pkgdown and test coverage workflows, hopefully that is Ok for the usethis case.